pkg/tar/tar_test.go

```diff
@@ -185,23 +187,60 @@ func TestCreateTar(t *testing.T) {
 	verifyTarFile(t, tarFile, testFiles, testLinks)
 }
 
-func createTestTar(files []fileDesc, writer io.Writer) {
+func createTestTar(files []fileDesc, writer io.Writer, t *testing.T) {
```

Comment:
As far I understand this method doesn't return `error` and when it fails it do it himself by calling `r.Fatalf()`. I don't understand why I need to return `error`? Could you explain, please?